app/scripts/controllers/certificates.js

```diff
@@ -108,6 +108,8 @@ angular.module('openhimConsoleApp')
       switch ( $scope.uploadType ){
         case 'serverCert':
           $scope.certificateObject.cert = data;
+          $scope.certificateObject.passphrase = $scope.serverPassphrase;
```

Comment:
This would be necessary for both, for usability's sake if you modify either of them you want the passphrase field sent in the request as well, I was avoiding having a seperate button